src/main/kotlin/no/nav/klage/oppgave/service/DokumentService.kt

```diff
@@ -106,10 +63,34 @@ class DokumentService(
         } ?: throw JournalpostNotFoundException("Journalpost $journalpostId not found")
     }
 
+    fun validateJournalpostExistsAsSystembruker(journalpostId: String) {
+        try {
+            safGraphQlClient.getJournalpostAsSystembruker(journalpostId)
+        } catch (e: Exception) {
+            logger.warn("Unable to find journalpost $journalpostId", e)
+            null
```

Comment:
Synes denne var litt vanskelig å lese. Vi skal jo ikke returnere noe her. Kunne man satt "throw" i catchen i stedet? Kommer vi catch skal vel alltid kaste videre?

Reply:
Jeg kan forstå at den er litt vanskelig å lese. Grunnen til at jeg ikke har throw i catchen er for å bare ha en throw. Metoden i safGraphQlClient kan jo returnere null, og da hiver jeg også en exception.

Reply:
Riktig. Ser den.